dmoj/executors/mixins.py

```diff
@@ -111,9 +111,11 @@ def get_address_grace(self):
             def get_env(self):
                 env = {'LANG': 'C'}
                 if self.unbuffered:
-                    unbuf = self._file('unbuf.so')
-                    shutil.copyfile(nobuf_path, unbuf)
+                    unbuf = self._file('setbufsize.so')
+                    shutil.copyfile(setbufsize_path, unbuf)
                     env['LD_PRELOAD'] = unbuf
+                    env['CPTBOX_STDOUT_BUFFER_SIZE'] = 0
```

Comment:
Might want to provide an interface to set this.

Reply:
Goal was to get nobuffer functionality working first